### PR TITLE
Use of the new model

### PR DESCRIPTION
**Pull request by benoitf**
- remove d.ts part of the plugin (use from the model)
 - webpack theia is packaged as external
 - for now the local @theia/plugin is still a local dependency as it is not yet published

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
   "description": "Simple Test Plugin",
   "main": "dist/bundle.js",
   "scripts": {
-    "start": "webpack-cli --config webpack.config.js",
+    "build": "webpack-cli --config webpack.config.js",
```
Comment (benoitf) [inline package.json]: I added "build" for building

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "webpack-cli": "^2.0.12"
   },
   "dependencies": {
+    "@theia/plugin": "file:///..../git/theia/packages/plugin",
```
Comment (benoitf) [inline package.json]: each ppl need for now to use yarn add file:/// (or I miss something ? )

Comment (evidolob) [reply package.json]: You are right, until we publish theia.d.ts into npm registry

Comment (benoitf) [reply package.json]: do you know if on theia side, some artifacts are published for PR/branch ?

**Pull request merged**